aspnetcore/fundamentals/routing.md

```diff
@@ -586,6 +586,79 @@ Complex segments (for example `[Route("/x{token}y")]`) are processed by matching
 <!-- While that code is no longer used by ASP.NET Core for complex segment matching, it provides a good match to the current algorithm. The [current code](https://github.com/aspnet/AspNetCore/blob/91514c9af7e0f4c44029b51f05a01c6fe4c96e4c/src/Http/Routing/src/Matching/DfaMatcherBuilder.cs#L227-L244) is too abstracted from matching to be useful for understanding complex segment matching.
 -->
 
+## Configuring endpoint metadata
+
+The following links provide information on configuring endpoint metadata:
+
+* [Enable Cors with endpoint routing](xref:security/cors#enable-cors-with-endpoint-routing)
+* [IAuthorizationPolicyProvider sample](https://github.com/aspnet/AspNetCore/tree/release/3.0/src/Security/samples/CustomPolicyProvider) using a custom `[MinimumAgeAuthorize]` attribute
+* [Test authentication with the [Authorize] attribute](xref:security/authentication/identity#test-identity)
+* <xref:Microsoft.AspNetCore.Builder.AuthorizationEndpointConventionBuilderExtensions.RequireAuthorization*>
+* [Selecting the scheme with the [Authorize] attribute](xref:security/authorization/limitingidentitybyscheme#selecting-the-scheme-with-the-authorize-attribute)
+* [Applying policies using the [Authorize] attribute](xref:security/authorization/policies#applying-policies-to-mvc-controllers)
+* <xref:security/authorization/roles>
+
+<a name="hostmatch"></a>
+
+## Host matching in routes with RequireHost
+
+`RequireHost` applies a constraint to the route which requires the specified host. The `RequireHost` or `[Host]` parameter can be:
+
+* Host: `www.domain.com` (matches `www.domain.com` with any port)
+* Host with wildcard: `*.domain.com` (matches `domain.com`, `www.domain.com`, `subdomain.domain.com`, or `www.subdomain.domain.com` on any port)
```

Comment:
I added some more tests to this area and a wildcard match must have a subdomain.

```suggestion
* Host with wildcard: `*.domain.com` (matches `www.domain.com`, `subdomain.domain.com`, or `www.subdomain.domain.com` on any port)
```